models/ga4_export__conversions_report.sql

```diff
@@ -1,20 +1,11 @@
 {{
     config(
-        enabled=var('key_events', none) is not none and var('key_events') != []
-    )
-}}
-
-{{
-    config(
+        enabled=var('key_events',[]) != [],
```

Comment:
These two configs are not necessary, we can consolidate them into one. Also, is there a reason we check if key_events is non and empty? If we set the default, then we can just check for that scenario.